### ISSUE TITLE
Newer Buildbot Version?

### ISSUE THREAD

**Issue by MTRNord** [claws/buildbot-prometheus#1]
Is that possible to use in newer versions of buildbot? Or do you have a list what would be needed to be updated?

**Comment by claws** [comment]
Yes, it should work in the latest version of buildbot. I built it for version 0.9. The current version is 0.9.5 so it should still work. Have you tried it?

**Comment by MTRNord** [comment]
not yet :) but when I have time for that I will report if it works

**Comment by Frodox** [comment]
I have tried and it didn't work, because it downgrade buildbot package! ;(
```
pip install buildbot_prometheus
...
Collecting buildbot==0.9.0rc3 (from buildbot_prometheus)
Successfully installed buildbot-0.9.0rc3 buildbot-prometheus-16.9.2 ....
```

while I have buildbot-0.9.9p2 before it
[Here is reason](https://github.com/claws/buildbot-prometheus/blob/8d2af714951dc81f52c0f4e2e0f06cb663c28abd/requirements.txt#L1)

**Comment by Frodox** [comment]
Take a look here https://github.com/tardyp/buildbot_profiler

**Comment by claws** [comment]
I missed that I had pinned the buildbot version to a specific release. I have updated the repo (see #2) and released a new version on pypi. This should resolve the problem.

**Comment by claws** [comment]
This issue was resolved in #2.
